### PR TITLE
Dev game buttons

### PR DESCRIPTION
**Pull request by Hboni**
Create area for in game buttons and create Quit button.

### CODE REVIEW

File: data/config/game.cfg.json [hunk 1]
```diff
@@ -1,7 +1,14 @@
 {
     "window": {
         "width": 1200,
-        "height": 700
+        "height": 700,
+        "game_buttons": {
```
Comment (deplanty) [inline data/config/game.cfg.json]: "game_buttons" should be outside of "window"

File: src/views/view_game.py [hunk 2]
```diff
@@ -34,6 +35,10 @@ def init_window(self):
 
         # Display on windows
         self.window.blit(self.background, (0, 0))
+
+        self.init_game_btns()
```
Comment (ldehaine) [inline src/views/view_game.py:39]: I'll definitely refactor this so buttons are updated the same way as Cards and Tokens, but that's ok FTM

File: src/views/view_game.py [hunk 3]
```diff
@@ -43,6 +48,21 @@ def init_window(self):
         dirty = all_sprites.draw(self.window)
         pygame.display.update(dirty)
 
+    def init_game_btns(self):
+        # Display game buttons area
+        cfg_btns = self.view_config['window']['game_buttons']
+        pygame.draw.rect(self.window, cfg_btns['color'],
+                         (cfg_btns['x'], cfg_btns['y'],
+                          cfg_btns['width'], cfg_btns['height']))
```
Comment (ldehaine) [inline src/views/view_game.py:56]: This is gonna be so much better when we can do cfg_btns = game_view_config.game_buttons, cfg_btns.x, cfg_btns.width...

File: src/views/view_game.py [hunk 4]
```diff
@@ -43,6 +48,21 @@ def init_window(self):
...
+
+        self.quit_btn = Button(pos=(1100, 610),
+                          width=80,
+                          height=80,
+                          text='Quit',
+                          background=(180, 180, 180),
+                          command=lambda x: print('a'))
+        self.quit_btn.display(self.window)
```
Comment (ldehaine) [inline src/views/view_game.py:64]: This is not related to this PR but I really don't like this.
self.window.blit(self.quit_btn, (x, y)) seems way more logical

File: src/controller/game_controller.py [hunk 5]
```diff
@@ -80,6 +80,10 @@ def playoneround(self):
                         return False
                     elif event.type == KEYDOWN and event.key == K_ESCAPE:
                         return False
+                    elif event.type == pygame.MOUSEBUTTONDOWN:
```
Comment (ldehaine) [inline src/controller/game_controller.py]: Don't we want something like MOUSEBUTTONDOWN then MOUSEBUTTONUP (click-release)?

**Review by ldehaine — CHANGES_REQUESTED** [review]

File: src/views/view_game.py (hunk 3 above)
Comment (Hboni) [reply src/views/view_game.py:56]: YEAAAAAAH ! I'm waiting for your PR to change it

File: src/views/view_game.py (hunk 4 above)
Comment (Hboni) [reply src/views/view_game.py:64]: `Button` class was implemented to have the possibility to do like that, and you don't have to give position, it uses directly `Button` attributes. For me `display` method is good !

File: src/controller/game_controller.py (hunk 5 above)
Comment (Hboni) [reply src/controller/game_controller.py]: Mmmmmh, maybe we can use MOUSEBUTTONUP only instead ?

File: data/config/game.cfg.json (hunk 1 above)
Comment (Hboni) [reply data/config/game.cfg.json]: Pushed !

**Review by deplanty — APPROVED** [review]
C b1

File: src/controller/game_controller.py (hunk 5 above)
Comment (ldehaine) [reply src/controller/game_controller.py]: OK

**Review by ldehaine — APPROVED** [review]

**Pull request merged**